src/encoded/static/components/mixins.js

```diff
@@ -572,9 +600,11 @@ module.exports.HistoryAndTriggers = {
             } else {
                 window.history.pushState(null, '', response_url + fragment);
             }
-            this.setProps({
-                href: response_url + fragment
-            });
+            // store.dispatch({
+            //     type: 'href',
+            //     value: response_url + fragment
+            // });
+            dispatch_dict['href'] = response_url + fragment;
```

Comment:
ok so we discussed this a bit in the meeting today, I guess lets go with codacy's convention and use . instead of []